schemas/subsets/main.yml

```diff
@@ -366,10 +356,6 @@ fields:
       tty:
         fields: "*"
       uptime: {}
-      user:
```

Comment:
Even if these won't be used right under `process.*`, with the way the re-uses render in the ECS docs, not having `process.user.*` exist at all could create confusion.

@kgeller WDYT?

Reply:
Yea, there is a bit of contention on whether we include process.user since endpoint has always used the top level user to track the user for the top level process, and wanted to keep it as such. I do agree this could create confusion. cc @ferullo @rickrobbins. Considering user is only an id and name field in this nesting, are we ok with populating both?

Reply:
I agree to populate the top-level `user.*` over `process.user.*`.

Leaving it in ECS just leaves the reuse to "exist" so when `process.*` is reused, it makes sense how that's chained together -> `process.*` contains `process.user.*` which is reused as `process.entry_leader.user.*`

Reply:
I agree with Eric, I think we keep its existence, even if there are no plans for endpoint to utilize it.

Reply:
Ok, i'll add this back in.